src/components/Splash.js

```diff
@@ -103,84 +145,142 @@ export default ({
+          </Link>
+          <Link
+            href={speakerFormLink}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <i className="fa fa-comment" style={{ fontSize: `1.2rem` }} />
+            <LinkText>Give a Talk</LinkText>
+          </Link>
+          <Link href="./p/code_of_conduct" target="_blank">
+            <i className="fa fa-sticky-note" style={{ fontSize: `1.2rem` }} />
+            <LinkText>Code of Conduct</LinkText>
+          </Link>
+          <Link href="./p/report" target="_blank">
+            <i className="fa fa-flag" style={{ fontSize: `1.2rem` }} />
+            <LinkText>Report Abuse</LinkText>
+          </Link>
+          <Link href="https://tldr.torontojs.com/" target="_blank">
+            <i className="fa fa-folder" style={{ fontSize: `1.2rem` }} />
+            <LinkText>Newsletter</LinkText>
+          </Link>
+        </Header>
+
+        {/* Toronto JS Logo Vector in Background */}
+        <InnerContainer>
+          <TitleContainer>
+            <Title>
+              {page}&nbsp;<b style={{ color: `#ffffff` }}>JS</b>
+            </Title>
+            <Rotator
+              items={[`MEETUPS`, `TECH TALKS`, `MOB PROGRAMMING`, `WORKSHOPS`, `SOCIAL EVENTS`]}
+            />
+          </TitleContainer>
+          <Tower fill={backgroundColor} />
+        </InnerContainer>
+      </Container>
+
+
+      {/* Main Content Section */}
+
+      <Container style={{ backgroundColor: `white`, lineHeight: 1.5, padding: `2rem 0` }}>
+
+        <InnerContainer style={{ display: `flex` }}>
+          <div style={{ width: `auto`, fontSize: `2rem` }}>
+            We are a volunteer-run community that supports all in their learning and passion for sharing knowledge on JavaScript- and by extension, software development in Toronto and the GTA.
+
+            <ul style={{ listStyle: `none`, fontSize: `1.25rem`, lineHeight: 1.4, marginTop: `1rem` }}>
+              <li>📅 Over 30+ events a year</li>
+              <li>🚀 Community-led online and in-person events</li>
+              <li>🦋 5555 on Slack</li>
+              <li>🚀 Over 400 on Guild</li>
+              <li>👯🏻 30+ <a href="./p/volunteers">volunteers 🧑🏻‍🤝‍🧑🏾</a></li>
+              <li>🎲 ~4 organizers</li>
+            </ul>
+          </div>
+
+          <iframe src="https://guild.host/embeds/guild/torontojs/card"
+            loading="lazy"
+            style={{ width: `450px`, height: `360px`, border: `none`, overflow: `hidden` }}
+          />
+
+        </InnerContainer>
+        <InnerContainer style={{ display: `flex`, justifyContent: `space-evenly` }}>
+          <div>
+            <Title>Upcoming Events</Title>
+            <iframe src="https://guild.host/embeds/guild/torontojs/events/upcoming"
+              loading="lazy"
+              scrolling="no"
+              frameBorder="0"
+              style={{ maxWidth: `350px`, minWidth: `300px`, minHeight: `360px`, border: `none`, overflow: `hidden` }}
```

Comment:
I've tweaked this a bit more in the preview 😅 

```suggestion
              style={{ width: `400px`, minHeight: `400px`, height: '100vh', border: `none`, overflow: `hidden` }}
```